posthog/queries/trends/sql.py

```diff
@@ -36,14 +36,25 @@
 )
 """
 
-ACTIVE_USER_SQL = """
-SELECT counts as total, timestamp as day_start FROM (
-    SELECT d.timestamp, COUNT(DISTINCT {aggregator}) counts FROM (
-        SELECT toStartOfDay(toTimeZone(toDateTime(timestamp, 'UTC'), %(timezone)s)) as timestamp FROM events WHERE team_id = %(team_id)s {parsed_date_from_prev_range} {parsed_date_to} GROUP BY timestamp
+ACTIVE_USERS_SQL = """
+SELECT counts AS total, timestamp AS day_start FROM (
+    SELECT d.timestamp, COUNT(DISTINCT {aggregator}) AS counts FROM (
+        -- We generate a table of periods to match events against. This has to be synthesized from `numbers`
```

Comment:
Note: Comments-in-sql are kind of problematic:
- There's a max query size limit we might bump up against
- It kills copying the queries from metabase which is important

Suggestions to fix:
- Move comment above the SQL to be python code comment
- Use block comment style, not `--`

Reply:
I see, thanks. I do think comments are a lot more useful next to the relevant code, so I'm wary of just piling them above the query in Python… For now I moved them to multiline `/* */` comments within the query.